src/System.Management.Automation/security/wldpNativeMethods.cs

```diff
@@ -113,9 +113,8 @@ public static SystemEnforcementMode GetLockdownPolicy(string path, SafeHandle ha
         private static SystemEnforcementMode GetWldpPolicy(string path, SafeHandle handle)
         {
             // If the WLDP assembly is missing (such as windows 7 or down OS), return default/None to skip WLDP validation
-            if (s_hadMissingWldpAssembly || !IO.File.Exists(IO.Path.Combine(Environment.SystemDirectory, "wldp.dll")))
+            if (s_hadMissingWldpAssembly)
```

Comment:
This is still needed for Operating System before Windows 10

Reply:
cc @PaulHigin

Reply:
Actually I feel this is the right fix.  We catch the *DllNotFoundException* and set the `s_hadMissingWldpAssembly` field there.  I don't see a need to check it every time at the top of the method.

Reply:
Yes, it is already in the PR description. 

I just do not understand why the security fix (mentioned in related issue) led to this problem if it is a long-standing code? Or is there still another problem?